generator/generator.yml

```
@@ -628,3 +628,11 @@ modules:
   hrSWInstalled:
     walk:
       - hrSWInstalled
+
+#
+# Schleifenbauer
+#
+# https://docs.schleifenbauer.eu/?dir=Interfaces/SNMP
+  schleifenbauer:
+    walk:
+      - 1.3.6.1.4.1.31034.12
```

This OID is overly broad. I think we want to be more targeted here with which walks to include. Some of the metrics included by this are read-write settings OIDs that have no meaning as metrics.

Let's start with the most useful part, the device table.

```suggestion
      - sdbMgmtCtrlDevicesTable
```

---

I think almost everything in 1.3.6.1.4.1.31034.12.1.1.2 would be of use, which is basically the whole generated list. There are a few you could leave out, but for my usecase more is better.